contract_api/consumers/service_event_consumer.py

```diff
@@ -112,6 +112,62 @@ def _get_new_assets_url(self, org_id, service_id, new_ipfs_data):
                                                                  service_id)
         return assets_url_mapping
 
+    def create_service_media(self,org_id,service_id,service_media,service_row_id,assets_url):
+        count = 0;
+        if(service_media!=None):
+            #get existing media data
+            existing_service_media_data = self._service_repository.get_service_media(service_id=service_id,org_id=org_id)
+
+            #delete existing media files if present form s3 bucket except video which is not saved
+            if existing_service_media_data!=None:
+                for media in existing_service_media_data:
+                    if(len(media['url'])>0 and media('file_type')!="video"):
+                        self._s3_util.delete_file_from_s3(url=media('url'))
```

Comment:
Why delete? When you delete here the website is broken till the new links are set up. We can handle delete in an archive job separately

Reply:
this is removed